### PR TITLE
Fix #14820: Table hide button in navigation

### LINKED ISSUE #14820: Hide buttons for tables is too obscure and close to the actual tables
### Is your feature request related to a problem? Please describe.

I just spend an hour trying to solve why my Laravel project was working **correctly** without a specific table in the database that should have been migrated properly. I felt like an idiot when I discovered it was hidden but then I noticed extremely obscure eyes next to the tables, only slightly visible when hovering over a table in the left navbar. When a folder is hidden the only way to notice, without knowing you hidden something, is a somewhat less obscure eye next to the database name. But for someone that doesn't know his table is hidden it's not the first place to look.

### Describe the solution you'd like

Remove hide button and put it somewhere else.

### Describe alternatives you've considered

Put it all the way to the right on the navbar or don't hide it completely when not hovering over a table. Since atm you can easily click it and the only way you know is that you are not redirected to the table you thought you clicked.

### Additional context

- phpMyAdmin version: 4.84

### PR DESCRIPTION
**Pull request by rabeehrz**
Signed-off-by: Mohammed Rabeeh <mohammadrabeeh@gmail.com>

### Description

Fix #14820 

### Fixes

Before Fix:
[](https://gifyu.com/image/qgHu)

After Fix:
[](https://gifyu.com/image/qgHn)

### CODE REVIEW

**Comment by rabeehrz** [comment]
@williamdes Please review and let me know if any changes are required. :)

**Review by williamdes — APPROVED** [review]
LGTM, I would maybe need another review

**Review by williamdes — CHANGES_REQUESTED** [review]
The eyes are always visible, can you fix this ?

**Comment by rabeehrz** [comment]
> The eyes are always visible, can you fix this ?

I initially thought that's how it was supposed to be. Going through issue once more clarified it. I've fixed it. :)

File: themes/pmahomme/scss/_navigation.scss [hunk 1]
```diff
@@ -157,23 +157,25 @@
...
-      display: inline;
+      display: block;
       opacity: 0.5;
 
       &:hover {
-        display: inline;
         opacity: 1;
       }
     }
+
```
Comment (williamdes) [inline themes/pmahomme/scss/_navigation.scss]: `yarn run css-lint --fix`

Will be okay this time :)

Comment (rabeehrz) [reply themes/pmahomme/scss/_navigation.scss]: My bad. Sorry. 😅

**Comment by williamdes** [comment]
Merged
Thank you for the improvement !

**Review by williamdes — APPROVED** [review]

File: themes/pmahomme/scss/_navigation.scss (hunk 1 above)
Comment (williamdes) [reply themes/pmahomme/scss/_navigation.scss]: no problem ;)

**Pull request merged**